addons/l10n_id_efaktur/models/efaktur.py

```diff
@@ -14,44 +14,55 @@ class Efaktur(models.Model):
     company_id = fields.Many2one('res.company', required=True, default=lambda self: self.env.company)
     max = fields.Char(compute='_compute_default', store=True, readonly=False)
     min = fields.Char(compute='_compute_default', store=True, readonly=False)
-    available = fields.Integer(compute='_compute_available', store=True)
+    available = fields.Integer(readonly=True)
+    next_num = fields.Integer(compute="_compute_next_num", store=True)
```

Comment:
```suggestion
    next_num = fields.Integer(compute="_compute_next_num", store=True)
```
I think we don't need to store this one